util/build.gradle

```diff
@@ -1 +1,11 @@
 description = 'Apollo Utilities'
+
+task(genRsa, dependsOn: classes, type: JavaExec) {
+    def utilSubproject = project(':util')
```

Comment:
Since we're in `util/` the current project can be referred to as `project` do this local var can be dropped.